tools/PI/DevHome.PI/Helpers/ExternalTool_v1.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json.Serialization;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace DevHome.PI.Helpers;
+
+public enum ExternalToolArgType
+{
+    None,
+    ProcessId,
+    Hwnd,
+}
+
+public partial class ExternalTool_v1 : ObservableObject
+{
+    public string ID { get; private set; }
+
+    public string Name { get; private set; }
+
+    public string Executable { get; private set; }
+
+    [JsonConverter(typeof(EnumStringConverter<ExternalToolArgType>))]
+    public ExternalToolArgType ArgType { get; private set; } = ExternalToolArgType.None;
+
+    public string ArgPrefix
+    {
+        get; private set;
+    }
+
+    public string OtherArgs
+    {
+        get; private set;
+    }
+
+    public bool IsPinned
+    {
+        get; private set;
+    }
```

Comment:
Nit to keep the file consistent
```suggestion
    public string ArgPrefix{ get; private set; }

    public string OtherArgs{ get; private set; }

    public bool IsPinned{ get; private set; }
```

Reply:
Thanks, fixed.